types/ComboQuery.md

```diff
@@ -0,0 +1,96 @@
+*Query Descriptor:*
+
+```json
+{
+  "type": "CrossChainBalance",
+  "inputs": [
+        {
+      "type": "bytes[]",
+      "name": "queryData"
+    },
+  ],
+  "outputs": [
+    {
+      "type": "bytes[]",
+      "packed": false
+    }
+  ]
+}
+```
+
+*queryData:* 
+
+Example for SpotPrice(ETH,USD) and SpotPrice(BTC,USD)
+
+```s
+keccak256(abi.encode("ComboQuery", abi.encode(
+                                      [abi.encode("SpotPrice", abi.encode("ETH","USD")),
+                                      abi.encode("SpotPrice", abi.encode("BTC","USD"))]
+)))
+```
+
+`0x00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000001c0000000000000000000000000000000000000000000000000000000000000016000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000953706f745072696365000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000003657468000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000037573640000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000016000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000953706f745072696365000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000003627463000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000037573640000000000000000000000000000000000000000000000000000000000`
+
+*queryID:*
+
+```s
+keccak256(abi.encode("ComboQuery", abi.encode(
+                                      [abi.encode("SpotPrice", abi.encode("ETH","USD")),
+                                      abi.encode("SpotPrice", abi.encode("BTC","USD"))]
+)))
+```
+
+`0xf22ce1368a35183b702c150a2a6d62d1c86fca00c64c44928e4bc7ba75fbd086`
+
+
+
+## Dispute Considerations
+
+- It is the reporters responsibility to view the dispute considerations for each queryId in the combo datatype
+
+
+## Example
+
+In the example query:  
+
+```
+abi.encode("ComboQuery", abi.encode(
+            [abi.encode("SpotPrice", abi.encode("ETH","USD")),
```

Comment:
SpotPrice args should be lower case: SpotPrice(eth, usd)

Reply:
fixed